docker/helper.Dockerfile

```diff
@@ -14,20 +13,7 @@ RUN set -eux; \
 FROM debian:bullseye-slim
 ENV HELPER_BIN_PATH=/usr/local/bin/ipa-helper
 ENV CONF_DIR=/etc/ipa
-ARG IDENTITY
-ARG HOSTNAME
 ARG SOURCES_DIR
 
 RUN apt-get update && apt-get install -y ca-certificates && rm -rf /var/lib/apt/lists/*
-COPY --from=builder ${SOURCES_DIR}/target/release/helper $HELPER_BIN_PATH
-
-# generate certificate/private key for TLS
-# make sure these names are consistent with the ones defined in CliPaths trait: src\cli\paths.rs
-RUN set -eux; \
```

Comment:
what's the motivation behind these changes?

Reply:
The idea is to use this docker inside the Kubernetes cluster. We want the key generation to happen before the Docker is run.